flixel/system/debug/Console.hx

```diff
@@ -112,7 +112,7 @@ class Console extends Window
 		_input.addEventListener(FocusEvent.FOCUS_OUT, onFocusLost);
 		_input.addEventListener(KeyboardEvent.KEY_DOWN, onKeyPress);
 		
-		#if (cpp || neko) // workaround for broken TextField focus on native
```

Comment:
There are two more code sections that use these conditionals, those should probably use `!next && sys` too?